app/models/manageiq/providers/hawkular/middleware_manager/middleware_server.rb

```diff
@@ -1,5 +1,9 @@
 module ManageIQ::Providers
   class Hawkular::MiddlewareManager::MiddlewareServer < MiddlewareServer
+    def feed
+      CGI.unescape(super)
```

Comment:
@cfcosta This breaks with..

```
> o.feed
NoMethodError: undefined method `tr' for nil:NilClass
```

when feed is `nil`. (Breaking ui-classic's specs.)

Can you perhaps add some logic to only unescape when non-nil?